src/main/java/seedu/address/logic/parser/AddLessonCommandParser.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.parser;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import seedu.address.logic.commands.AddLessonCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.lessons.Lesson;
+import seedu.address.model.person.Subject;
+
+
+
+/**
+ * Parses input arguments and creates a new AddLessonCommand object
+ */
+public class AddLessonCommandParser implements Parser<AddLessonCommand> {
+
+    // let be consistent with this, a model's optional field could be null, but during the rendering we check for that
```

Comment:
not sure what you meant by this comment

Reply:
You are right, using null as default value is proven to be a bad idea, and I make an update in the next pr that I am going to push. You could safely ignore this line now.

Basically I added a static default value for phone, address, name and email, so that the user is not false to input all fields, and can edit at anytime they want. I also fix the dependency with them